source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs

```diff
@@ -94,7 +94,18 @@ public void TestParsingInvalidCardMissingVersion()
     }
   ]
 }";
+            // By default we should be throwing an exception when no version present.
             Assert.ThrowsException<AdaptiveSerializationException>(() => AdaptiveCard.FromJson(json));
```

Comment:
Nit: can you add a test message with the assert call assuming it allwos for one for easier debugging?

Reply:
this is just a unit test :)

---
In reply to: [325817469](https://github.com/microsoft/AdaptiveCards/pull/3462#discussion_r325817469) [](ancestors = 325817469)

Reply:
Yes i realize that :) I'd love to have use the same standard for test files as we do for prod -- debugging test failures in teh past, ive found tests that include specific strings/messgaes that can help me parse test failures without neding to plop open the source files go a long way! 

---
In reply to: [325824969](https://github.com/microsoft/AdaptiveCards/pull/3462#discussion_r325824969) [](ancestors = 325824969,325817469)

Reply:
Oh! I see what you're saying. I misunderstood. :)

---
In reply to: [325828828](https://github.com/microsoft/AdaptiveCards/pull/3462#discussion_r325828828) [](ancestors = 325828828,325824969,325817469)